ixblue_ins_driver/src/ros_publisher.cpp

```diff
@@ -461,3 +470,19 @@ ROSPublisher::toiXInsMsg(const ixblue_stdbin_decoder::Data::BinaryNav& navData)
 
     return res;
 }
+
+ixblue_ins_msgs::ExternalSensorDataPtr
+ROSPublisher::toiXExternalSensorMsg(const ixblue_stdbin_decoder::Data::BinaryNav& navData) {
+  // --- Check if there are enough data to send the message
+  if(navData.soundVelocity.is_initialized() == false) {
+      return nullptr;
+  }
+
+  // --- Initialisation
```

Comment:
Initialization

Reply:
It is not incorrect and I prefer with with z too. It is a UK english and ixblue guys use this way in other parts of the code.